tests/dummy/app/states/todos.js

```diff
@@ -1,6 +1,8 @@
 import BufferedProxy from 'ember-buffered-proxy/proxy';
 
-BufferedProxy.reopenClass({
+const TodosProxy = BufferedProxy.extend();
+
+TodosProxy.reopenClass({
   initialState() {
     return {
       content: this.get('todo')
```

Comment:
> It might be better/more straight forward to pass it in as an argument. Thoughts?

yes, I believe argument is best. The `this` in that method should actually be the class itself, where it should have access to other static methods.